tools/star.wdl

```diff
@@ -590,7 +590,7 @@ task alignment {
         Int winBinNbits = 16
         Int winAnchorDistNbins = 9
         Int winFlankNbins = 4
-        Int chimSegmentMin = 0
+        Int chimSegmentMin = 10
```

Comment:
@claymcleod and @a-frantz 

I've migrated all of the other overrides up to `rnaseq-core`. I've left these two as `chimSegmentMin` must be > 0 to enable chimeric detection, so I suspect we meant to enable that originally. For `chimOutType`, `WithinBAM` is recommended by the STAR manual (even though `Junctions` is the default).

Reply:
Just curious, why `10`?

IMO `chimOutType WithinBAM` is the right call. I think it's better to output chimeric alignments by default than to exclude them. That would necessitate having a `chimSegmentMin > 0`. (Weird set-up from STAR.) I'm just not sure where `10` came from.

Reply:
Valid question. I took `10` from Arriba. I don't know if that is a completely sane choice. I don't think allowing a chimeric segment to be too short (e.g. `1`bp) is wise. Below a certain value, you're not getting any meaningful information and likely just spurious alignments.